assets/css/style.css

```diff
@@ -309,3 +307,110 @@ hr {
   text-align: center;
   color: #6070ff;
 }
+
+@media (min-width: 768px) {
+  /* header */
+  header {
+    height: 73px;
+  }
+
+  header .container img {
+    display: none;
+  }
+
+  header .container ul {
+    display: flex;
+    gap: 20px;
+    color: #344563;
+  }
+
+  #hero {
+    background: url("../images/Header\ bg.svg") no-repeat #fff 100%;
+    border-bottom-left-radius: 160px;
+    background-position: center;
+    background-size: contain;
+    padding: 50px 140px;
+  }
+
+  /* card */
+  #work {
+    margin-top: 104px;
+  }
+
+  .work {
+    padding: 0 20px;
+  }
+
+  .card {
+    flex-direction: row;
+  }
+
+  .card .card-img {
+    width: 100%;
+    padding-right: 24px;
+  }
+
+  /* About */
+  #about .container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    padding: 80px 25px;
+  }
+
+  .skills .language {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+  }
+
+  .language li {
+    display: flex;
+    justify-content: start;
+    flex-direction: column;
+    align-items: start;
+    background-color: #f7f7f9;
+    border-radius: 5px;
+    text-decoration: none;
+    color: #253858;
+    padding: 5px 20px 0 10px;
+    font-size: 15px;
+  }
+
+  .contact {
+    background-image: url("../images/contact-bg.svg");
+    background-position: 100px cover;
+    background-size: auto;
+    background-repeat: no-repeat;
+    border-top-left-radius: 160px;
+    background-color: #6070ff;
+    padding: 50px 200px;
+  }
+
+  .contact input[type="text"],
+  .contact input[type="email"] {
+    border: 0;
+    border: 1px solid #fff;
+    width: 100%;
+    height: 48px;
+    font-size: 16px;
+    display: flex;
+    padding-left: 15px;
+  }
```

Comment:
- [ ] I would kindly suggest you stick with the Figma design as much as possible when designing your desktop version of the project. Currently, your contact section of the project does not look like what we have in the Figma design. Kindly reduce the width of the form and make it properly aligned.

Possible Implementation 👇 

  - Kindly reduce the width in the  class `.contact input[type="text"], .contact input[type="email"]`, and `.contact textarea` from `100%` to `50%` and add a margin of `0 auto`
  - Also kindly add this class `.contact .contact-text p` in your `media-query` for desktop design and give it the below properties

```CSS
.contact .contact-text p {
    font-style: normal;
    font-weight: 400;
    font-size: 20px;
    line-height: 28px;
    color: #ebebff;
    width: 70%;
    margin: 15px auto;
}
```


- [ ] Also i would kindly suggest you give all the buttons some effect as seen in the Figma design `i.e hover, active, enabled` etc.